keps/2724-topology-aware-scheduling/README.md

```diff
@@ -648,11 +653,13 @@ For a given PodSet Kueue:
   level. Kueue starts the search from the specified level, but if the PodSet
   does not fit, then it tries higher levels in the hierarchy.
 
-Kueue packs pods on domains starting from the domains with the most free capacity. However, Kueue can operate in two modes when it comes to choosing the last domain if there is more than one capable of accommodating the remaining pods:
+Kueue packs pods on domains starting from the domains with the most free capacity. However, Kueue can operate in three modes when it comes to choosing the last domain if there is more than one capable of accommodating the remaining pods:
 - `MostAllocated` - Kueue chooses the domain with the least available resource that is capable of accommodating all the pods to mitigate resource fragmentation
 - `LeastAllocated` - Kueue chooses the domain that has the most available resources, providing better nodes utilization
+- `ConditionalBestFit` - Kueue chooses the domain with the least available resources. It promotes minimizing resource fragmentation, and higher node utilization over compact placement. This algorithm is used for PodSet with the `kueue.x-k8s.io/podset-relaxed-topology: true` annotation set.
```

Comment:
tbh I dont get neither the Conditional not BestFit parts of the name :) . Maybe it should be called 
LeastFreeCapacityFirst?

and to be fair the current names are not good either. I would call them 'MostFreeCapacityFirst' and 'BestFit'.

Reply:
> Are you proposing:
> MostAllocated -> BestFit
> LeastAllocated -> MostFreeCapacityFirst
> ConditionalBestFit -> LeastFreeCapacityFirst

That's right.

Reply:
Done